src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -65,6 +69,7 @@ internal class BindingProcessImpl : IBindingProcess
             this.bindingConfigProvider = bindingConfigProvider ?? throw new ArgumentNullException(nameof(bindingConfigProvider));
             this.bindingMode = bindingMode;
             projectToLanguageMapper = host.GetMefService<IProjectToLanguageMapper>();
+            this.threadHandling = threadHandling ?? new ThreadHandling();
```

Comment:
If the class goes back to being MEF-importable, this will be a bit unclean, but I looked at the chain and it's ugly all the way up anyway (BindingController.CreateBindingProcess takes some from the host and some are new-ed up) so I guess we can just leave it like that.